docs/develop/go/test-suites.mdx

```diff
@@ -160,6 +161,103 @@ When an Activity sends a Heartbeat, be sure that you can see the Heartbeats in y
 
 If an Activity is supposed to react to a Cancellation, you can test whether it reacts correctly by canceling it.
 
+## Mock and override Nexus operations
+
+A workflow that executes a Nexus operation can be tested by providing a mock of the Nexus operation
```

Comment:
workflow should be uppercased. If you use Vale or throw this into a Vale-supported editor or IDE (I use Sublime, also works with VSCode), they are automatically highlighted and you can hover over anything and see why it was highlighted. Caution: the signal to noise ratio isn't great.

Alternatively, you can ask us to do an edit pass after content is complete and technically reviewed. Specify your comfort level of how much we can reword.

That is independent of this read-through, which I'm doing for flow and as a reader.